packages/shared/src/components/modals/report/ReportModal.tsx

```diff
@@ -34,6 +34,12 @@ export function ReportModal({
     onClick: (e) => onReport(e, reason, note),
   };
 
+  const onFocus = useCallback(() => {
+    if (!reason) {
```

Comment:
We need an additional check if the reason is empty (the user hasn't selected anything) as they may just want to add more notes.

Reply:
@sshanzel There is already a check for checking whether the reason is empty,  if it is empty then I am setting it to others

Reply:
Ah, nice. Yeah, I see it now. Didn't get much chance to see which `reason` was this, turns out to be the state that manages it. Well done! 🚀

Reply:
One last thing is, to remove the `useCallback`. We shouldn't need it as far as I see.